### PR TITLE
Feature/chat server interaction fixes #10

### LINKED ISSUE #10: Send requests to the backend.
The static room and song selection should be retrieved from the backend. This should post a request to the backend according to the APIs of the Core.

### PR DESCRIPTION
**Pull request by GijsWeterings**

### CODE REVIEW

File: app/scripts/controllers/main.js [hunk 1]
```diff
@@ -8,24 +8,64 @@
...
+      // // Store the data-dump of the FORM scope.
+      request.success(
+        function(response) {
+          console.log(angular.toJson(response));
+
+        }
+      );
+      request.error(
+        function() {
+          alert("Failed to send message!");
```
Comment (JWGmeligMeyling) [inline app/scripts/controllers/main.js]: Please use the `$log` service for logging. Console breaks on some browsers and alert is not that friendly. `$log` has methods for logging at various levels: `$log.warn` and `$log.info`.

You can also inline this:

```js
request.error($log.warn.bind($log, "Failed to send message!"));
``

**Comment by JWGmeligMeyling** [comment]
PR is ok but we should have a look at the disabled test cases :+1:

**Comment by JWGmeligMeyling** [comment]
Fixed the test with @GijsWeterings . PR requires https://github.com/MoodCat/MoodCat.me-Core/pull/65 . PR can be merged @JeremybellEU

**Comment by Azkle** [comment]
:+1:

**Pull request merged**